nengo_gui/exec_env.py

```diff
@@ -117,6 +117,11 @@ def __exit__(self, exc_type, exc_value, traceback):
 
         sys.stdout = sys.__stdout__
 
+        # ensure what has been printed is safe to show in html
+        s = self.stdout.getvalue()
+        s = s.replace('<', '&lt;').replace('>', '&gt;')
```

Comment:
Doesn't this part make the link not work? Like, you want to keep the `<` in what you pass to `.html` so that it renders as an actual link. Or is that not how `.html` works?